### ISSUE TITLE
two api need

### ISSUE THREAD

**Issue by universea** [PaddlePaddle/Paddle#18156]

**Comment by universea** [comment]
Now I am rewring pysot (Tracking models) by paddlepaddle ,but paddle don't have tensor.view() ,this api is like reshape,but they diffenent,we can look these difference in torch API.
Another api is conv2d ,but input are two images,the one is input ,the other is filter,we can look it in torch API torch.nn.functional.conv2d(input, weight, bias=None, stride=1, padding=0, dilation=1, groups=1)
